### DISCUSSION TITLE
ko/models/yolov8/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#8028] [Docs]
# ko/models/yolov8/

실시간 물체 감지기의 최신 버전인 YOLOv8 의 놀라운 기능을 살펴보세요! 고급 아키텍처, 사전 학습된 모델, 정확도와 속도 사이의 최적의 균형이 어떻게 YOLOv8 을 물체 감지 작업에 완벽한 선택으로 만드는지 알아보세요.

https://docs.ultralytics.com/ko/models/yolov8/

**Comment by dongsig2** [comment]
In YOLOv8, when training with the option 'Pretrained = True', is the pretrained dataset the COCO dataset? Or was it pretrained on a different dataset?

**Reply by glenn-jocher** [reply]
@dongsig2 when you use the option `Pretrained = True` in YOLOv8, the model utilizes weights that have been pretrained on the COCO dataset. This dataset is a large-scale object detection, segmentation, and captioning dataset widely used for training and benchmarking object detection models. Using these pretrained weights can help improve your model's performance, especially when you have a limited amount of training data. 🚀

**Reply by dongsig2** [reply]
Thank you :)

**Reply by pderrenger** [reply]
@dongsig2 you're welcome! If you have any more questions or need further assistance, feel free to ask. Happy coding with YOLOv8! 😊👍

**Comment by YeapChweap** [comment]
Hello! I'm using yolov8 and I just can't use it from CLI.

I have installed ultralytics as below
>>> ultralytics.checks()
Ultralytics YOLOv8.2.28 🚀 Python-3.11.9 torch-2.2.1+cu118 CUDA:0 (NVIDIA GeForce RTX 3060 Laptop GPU, 6144MiB)
Setup complete ✅ (16 CPUs, 15.3 GB RAM, 210.1/237.3 GB disk)

but everytime when I want to use the yolo command, it appears like this.

'yolo' is not recognized as an internal or external command operable program or batch file

**Reply by glenn-jocher** [reply]
@YeapChweap hello! 👋

Thank you for reaching out and providing detailed information about your setup. It looks like you're encountering an issue with the `yolo` command not being recognized in your CLI.

Here are a few steps to help resolve this:

1. **Ensure Installation Path**: Make sure that the `ultralytics` package is installed in your current environment and that the `yolo` command is available in your system's PATH. You can verify this by running:
   ```bash
   pip show ultralytics
   ```
   This should display the installation details of the `ultralytics` package.

2. **Reinstall Ultralytics**: Sometimes, reinstalling the package can resolve path issues. Try uninstalling and then reinstalling `ultralytics`:
   ```bash
   pip uninstall ultralytics
   pip install ultralytics
   ```

3. **Check Environment Variables**: Ensure that the directory containing the `yolo` executable is included in your system's PATH environment variable. You can add it manually if it's missing.

4. **Use Full Path**: As a temporary workaround, you can try using the full path to the `yolo` executable. You can find the path by running:
   ```bash
   which yolo
   ```
   or
   ```bash
   where yolo
   ```
   depending on your operating system.

5. **Update Packages**: Ensure you are using the latest versions of `torch` and `ultralytics`. You can update them using:
   ```bash
   pip install --upgrade torch ultralytics
   ```

If the issue persists, please provide a minimum reproducible example of your setup and commands. This will help us investigate further. You can find more details on creating a minimum reproducible example here: [Minimum Reproducible Example](https://docs.ultralytics.com/help/minimum_reproducible_example).

Feel free to share any additional details or screenshots if you encounter further issues. We're here to help! 😊
